### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by AlexGhiondea**
This is used to ensure that PRs are automatically labeled.

The format is that on the line above the codeowner entry, add a comment with this format:
`# PRLabel: %Label`

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -9,45 +9,60 @@
...
 /sdk/cognitiveservices/vision.computervision/   @toothache
 /sdk/cognitiveservices/vision.face/             @TFR258 
 
+# PRLabel: %DigitalTwins
 /sdk/digitaltwins/                              @drwill-ms @timtay-microsoft @abhipsaMisra @vinagesh @azabbasi @prmathur-microsoft @bikamani @barustum
 
+# PRLabel: %Event Grid
 /sdk/eventgrid/                                 @Kishp01 @ahamad-MS
 
+# PRLabel: %Event Hubs
```
Comment (jsquire) [inline .github/CODEOWNERS:35]: Not your addition, but I don't think we need line 37;  I don't remember why we started down that path, but line 36 covers that.

**Review by jsquire — APPROVED** [review]
LGTM

**Review by weshaggard — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (AlexGhiondea) [reply .github/CODEOWNERS:35]: Fixed.

**Pull request merged**